subprojects/core/build.gradle.kts

```diff
@@ -144,11 +145,11 @@ dependencies {
     testFixturesImplementation(libs.ant)
     testFixturesImplementation(libs.groovyAnt)
     testFixturesImplementation(libs.asm)
-
```

Comment:
🤔 How comes this needs to become `testFixturesImplementation` instead of `testFixturesRuntimeOnly`?

Reply:
added "because"